indico/modules/events/registration/util.py

```diff
@@ -185,3 +185,17 @@ def generate_csv_from_registrations(registrations, regform_items, special_items)
         writer.writerow(registration_dict)
     buf.seek(0)
     return buf
+
+
+def get_user_complete_registrations(user, event):
```

Comment:
not used except in the next function. i'd get rid of it and merge it into that function. we can always extract it if we ever need it without the ticket logic